packages/DatePicker/src/components/Calendar/Calendar.js

```diff
@@ -68,20 +128,32 @@ function Calendar(props) {
 
   // eslint-disable-next-line react/prop-types
   function renderMonthHeaderElement({ month }) {
-    return <CalendarHeaderStyled>{month.format(I18n.t("dateInput.calendar_header_format"))}</CalendarHeaderStyled>;
+    return (
+      <MonthHeaderButtonStyled
+        isDropdown
+        kind="flat"
+        onClick={() => {
+          handleClickHeader(month);
+        }}
+        tabIndex={-1}
+        data-qa-anchor="datepicker.calendar.header"
+      >
+        {month.format(I18n.t("datePicker.calendar_header_format"))}
+      </MonthHeaderButtonStyled>
+    );
   }
 
   function renderArrowLeft() {
     return (
-      <span ref={prevButtonRef} data-qa-anchor="datepicker-prev-month">
+      <span css={ArrowIconStyles} ref={prevButtonRef} data-qa-anchor="datepicker-prev-month">
         <ArrowLeft role="presentation" size="14px" />
```

Comment:
Incidental, but the `size` prop can be omitted, and `fontSize(-1)` can be used instead in the `ArrowIconStyles`.